api-tests/src/test/resources/org/openmrs/module/reporting/include/ReportTestDataset-openmrs-2.3.xml

```diff
@@ -0,0 +1,564 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
Do you remember why this 2.3 version is needed?  Were there changes since the previous version, or is it just a practice that we have one for each supported version?

Reply:
So, this is just following the standard practice. A number of tests have this in their setup:

```java
executeDataSet(XML_DATASET_PATH + new TestUtil().getTestDatasetFilename(XML_REPORT_TEST_DATASET));
```

Which turns into "ReportTestDataset-openmrs-<openmrs short version>.xml". Otherwise, this is identical to the 2.2 version.